content/docs/command-reference/add.md

```diff
@@ -24,15 +24,22 @@ into <abbr>data artifacts</abbr> of the <abbr>project</abbr>. By default, these
 are committed to the <abbr>cache</abbr> (use the `--no-commit` option to avoid
 this, and `dvc commit` to finish the process when needed).
 
-Note that [external data](/doc/user-guide/managing-external-data) (targets
+`dvc add` is typically used to version control raw data or initial datasets from
```

Comment:
I don't think it's a good idea. `dvc add model.pkl` is a very good use case by itself for those who don't care about pipelines. We should never restrict it any way. Instead we should _sell_ pipelines as additional feature - you can "understand" _how_ a specific artifact is built.

Also, it's _very_ common that _raw_ data versioning is not a good use case for DVC. e.g. it might be very large and does not fit onto a local machine as a whole.

Also, _raw_ can be considered like real _raw_ data that most companies use Hadoop/Spark to deal with - it's a BigData/ETL world.

Reply:
where /why does this comment come from, btw?

Reply:
I see that it comes from the bottom of the `dvc add` ... not sure what is the history of that, but we should probably change the wording a bit

Reply:
Yeah I was just reordering the paragraphs. OK, updating... ⏳